modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -167,6 +170,14 @@ public class ObjectEntryLocalServiceTest {
 			new LiferayIntegrationTestRule(),
 			SynchronousDestinationTestRule.INSTANCE);
 
+	@BeforeClass
+	public static void setUpClass() throws PortalException {
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext();
+
+		_originalWorkflowAction = serviceContext.getWorkflowAction();
+	}
```

Comment:
Can you add this in `setUp`? If we add a `@BeforeClass`. We need create a `@AfterClass`.

Reply:
The `setUp` runs before every test. So, from my perspective, it wouldn't make sense to get the _original_ value there. 
Also, in the [example class](https://github.com/liferay/liferay-portal/blob/fad61814ed933990b39b7f2744feb0e6cac549ba/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/related/models/test/ObjectRelatedModelsProviderTest.java#L95-L96) you provided, there is a `@BeforeClass`, without `@AfterClass`. The way I see it, that is okay. Do you think this is a problem?